products/kms/api.yaml

```diff
@@ -33,8 +33,7 @@ objects:
     input: true
     description: |
       A `KeyRing` is a toplevel logical grouping of `CryptoKeys`.
-    collection_url_response: !ruby/object:Api::Resource::ResponseList
-      items: 'keyRings'
+    collection_url_key: 'keyRings'
```

Comment:
`collection_url_keys` now defaults to the camelcased, plural version of the name. In theory, this line should be unnecessary.

(I'm using ActiveSupport's `pluralize` to make words plural and it's super smart but I'm always hesitant of it because it's black magic)